language-specification.md

```diff
@@ -0,0 +1,141 @@
+# Language specification
+
+One aim of Swinebouk is to specify a language that can describe a vowel chart.
+
+## Requirements
+
+It must be text-based.
+
+It should be easy to write.
+
+It does not assume the symbol for a particular vowel. Some reasons:
+
+- Some vowels do not have dedicated symbols e.g. [the open central vowel](https://en.wikipedia.org/wiki/Open_central_vowel).
+- In broad transcription, one may use e.g. the symbol for a close-mid front unrounded vowel (e) to indicate a mid front unrounded vowel (e̞).
+
+It must be able to describe the position of a vowel: openness (y-axis), frontness (x-axis), roundness (left of the point, unrounded; right of the point, rounded).
+
+⚠️ What other qualities need to be described? [R-colouring](r-colouring)? [Vowel length][vowel-length]?
+
+[r-colouring]: https://en.wikipedia.org/wiki/R-colored_vowel
+[vowel-length]: https://en.wikipedia.org/wiki/Vowel_length
+
+## Solution Proposals
+
+### Rough idea 1: url query-like
+
+Chart with `i` and `u`:
+
+```
+front+close+unrounded=i&back+close+rounded=u
+```
+
+The articulation could alternatively be represented with abbreviations e.g.
+
+```
+fcu=i&bcr=u
+```
+
+Diphthongs/triphthongs can be represented with an arrow `->`. E.g. for the diphthong "au̯":
+
+```
+front+open+unrounded->back+close+rounded=au̯
+fou->bcr=au̯
+```
+
+⚠️ Is a dash (`-`) a better choice to an arrow?
```

Comment:
I would definitely say so! However, usually the `+` sign translates to a space in URLs, not sure about the meaning then...